.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Submitting Pull Requests
+We use [Waffle.io](https://waffle.io/ExpressGateway/express-gateway) to track work on the task board. Waffle uses keywords just like Github.
+
+Please see the checklist below and thanks again for your contribution!
+
+- [ ] please use "Connect #<Github Issue>" anywhere within the description
+    example) "fixes bug with OAuth2 - connect #345" 
```

Comment:
Were you intending this to be on a new line?  You might have to add an extra linebreak.  It shows up like:

> please use "Connect #<Github Issue>" anywhere within the description example) "fixes bug with OAuth2 - connect #345"

Reply:
yeah new line but "inline" with the bullet indendation sans the bullet itself, thx